awx/main/models/credential/__init__.py

```diff
@@ -1257,3 +1307,72 @@ def create(self):
         }
     },
 )
+
+
+class CredentialInputSource(PrimordialModel):
+
+    class Meta:
+        app_label = 'main'
+        unique_together = (('target_credential', 'input_field_name'),)
+
+    target_credential = models.ForeignKey(
+        'Credential',
+        related_name='input_source',
```

Comment:
`'input_source'` -> `'input_sources'`

It's almost impossible to fix these after we get the code merged, so I'd like to get them fixed in review.

consider `input_source for input_source in credential.input_source.all()`

vs.

`input_source for input_source in credential.input_sources.all()`

Reply:
Good 👁